src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
 
+        Machine[] machines = new Machine[3];
+        machines[0] = truck;
+        machines[1] = excavator;
+        machines[2] = bulldozer;
```

Comment:
we can also initialize array with our machines (`Machine[] machines = {truck, excavator, bulldozer};`)